cmd/root.go

```diff
@@ -63,18 +66,45 @@ func init() {
 	rootCmd.PersistentFlags().StringVar(&kubeconfig, "kubeconfig", "", "kubeconfig file path. This is optional, in cluster config will be used if not set")
 }
 
+// See: https://github.com/spf13/viper/issues/188#issuecomment-399884438
```

Comment:
This code is copied from a GitHub comment; what license is it under? We may need to rewrite it instead. (Also, I find the variable names to be opaque.)

Reply:
Right, but otoh it won't get much different if we end up using reflection. I'll try a different approach then